lmp/bb-config.sh

```diff
@@ -91,6 +92,7 @@ LMP_FACTORY_IMAGE = "${IMAGE}"
 SOTA_PACKED_CREDENTIALS = "${SOTA_PACKED_CREDENTIALS}"
 OSTREE_BRANCHNAME = "${MACHINE}-${OSTREE_BRANCHNAME}"
 GARAGE_TARGET_VERSION = "${H_BUILD}"
+GARAGE_TARGET_EXPIRE_AFTER = "${TUF_TARGETS_EXPIRE}"
```

Comment:
everything looks fine to me. @ricardosalveti will adding this variable cause any weird problems with sstate cache?

Reply:
```
Parsing of 3555 .bb files complete (0 cached, 3555 parsed). 5278 targets, 937 skipped, 9 masked, 0 errors.
Loading cache...done.
Loaded 5276 entries from dependency cache.
Parsing recipes...done.
Parsing of 3555 .bb files complete (3554 cached, 1 parsed). 5278 targets, 937 skipped, 9 masked, 0 errors.
(2023-01-18 15:18:50,904) INFO: Resolving any missing task queue dependencies
(2023-01-18 15:18:58,785) Level 21: 
```

Reply:
https://ci.foundries.io/projects/msul-dev01/lmp/builds/1701/intel-corei7-64/artifacts/console.log